src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -186,7 +186,16 @@ private static async Task<IReadOnlyList<ProposedEdit>> GetProposedEditsAsync(
             ITextSnapshot oldSnapshot, string? indentText, CancellationToken cancellationToken)
         {
             var list = new List<ProposedEdit>();
-            var (documentationCommentDictionary, isQuotaExceeded) = await copilotService.GetDocumentationCommentAsync(proposal, cancellationToken).ConfigureAwait(false);
+            var documentationCommentResults = await copilotService.GetDocumentationCommentAsync([proposal], cancellationToken).ConfigureAwait(false);
+
+            Dictionary<string, string>? documentationCommentDictionary = null;
+            var isQuotaExceeded = false;
+
+            if (!documentationCommentResults.IsEmpty)
+            {
+                documentationCommentDictionary = documentationCommentResults[0].responseDictionary;
+                isQuotaExceeded = documentationCommentResults[0].isQuotaExceeded;
+            }
```

Comment:
```suggestion
            var documentationCommentDictionary = documentationCommentResults.FirstOrNull()?.responseDictionary;
            var isQuotaExceeded = documentationCommentResults.FirstOrNull()?.isQuotaExceeded ?? false;
```

Reply:
can't deconstruct it

Reply:
How?  The signature is: `ImmutableArray<(Dictionary<string, string>? responseDictionary, bool isQuotaExceeded)>`, so FirstOrDefault will give you a `(Dictionary<string, string>? responseDictionary, bool isQuotaExceeded)` which should be deconstructible, right?

Reply:
you're right I must have been deconstructing the wrong thing yesterday